episodes/chapter6.md

```diff
@@ -0,0 +1,570 @@
+
+Each organization that is involved in a federated learning collaboration has its own node in that collaboration. They should therefore install the node software on a virtual machine hosted in their own infrastructure. The node should have access to the data that is used in the federated learning collaboration.
+
+This chapter will explain how to set up and run the vantage6 node software.
+
+
+## Requirements on hardware and software
+
+### Hardware requirements
+
+The minimal hardware requirements are:
+
+- x86 CPU architecture + virtualization enabled
+- 1 GB memory
+- 50GB+ storage
+- Stable and fast Internet connection (1 Mbps+)
+
+The hardware requirements of vantage6 node also depend on the algorithms that the node will run. For example, you need much less compute power for a descriptive statistical algorithm than for a machine learning model.
+
+
+### Software requirements
+
+The following software must be installed before installing the vantage6 node:
+
+- Operating system: Ubuntu 18.04+ , MacOS Big Sur+, or Windows 10+
+- Python
+    - Python v3.10 for vantage6 version 3.8.0 or higher
+    - Python v3.7 for other lower versions of vantage6
+    - Highly recommended to create a new, clean virtual or conda environment for vantage6 node
+- Docker (always latest version)
+
+::: callout
+## ⚠️ Docker installation 
+  - For Linux users, some [post-installation steps](https://docs.docker.com/engine/install/linux-postinstall/) may be required. Vantage6 needs to be able to run docker without sudo, and these steps ensure just that.
+
+  - For Windows users, if you are using Docker Desktop, it may be preferable to limit the amount of memory Docker can use - in some cases it may otherwise consume much memory and slow down the system. This may be achieved as described [here](https://stackoverflow.com/questions/62405765/memory-allocation-to-docker-containers-after-moving-to-wsl-2-in-windows).
+:::
+
+## Installation
+
+The Python package `vantage6` provides a command-line interface (CLI) to manage vantage6 infrastructure.
+
+To install this CLI package, run the command in your Python environment,
+
+```bash
+# First go to your python virtual environment
+# Then install the package
+
+pip install vantage6
+```
+
+To verify the installed CLI, run the command,
+
+```bash
+v6
+```
+or
+```bash
+v6 --help
+```
+
+If the installation is successful, it will print out the following messages,
+```bash
+
+Usage: v6 [OPTIONS] COMMAND [ARGS]...
+
+  The `v6` command line interface allows you to manage your vantage6
+  infrastructure.
+
+  It provides a number of subcommands to help you with this task.
+
+Options:
+  --help  Show this message and exit.
+
+Commands:
+  algorithm  Manage your vantage6 algorithms.
+  dev        Quickly manage a test network with a server and several nodes.
+  node       Manage your vantage6 node instances.
+  server     Manage your vantage6 server instances.
+```
```

Comment:
We should definitely report the complete list, to avoid any possible confusion. Maybe we can put the list as a solution to the challenge. It makes also the text a bit more readable.